build.gradle

```diff
@@ -169,8 +177,15 @@ dependencies {
 
   compile group: 'com.microsoft.azure', name: 'azure-servicebus', version: '1.2.7'
 
-  testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: versions.springBoot
+  annotationProcessor group: 'org.springframework.boot', name: 'spring-boot-configuration-processor'
+
```

Comment:
extra line here?

Reply:
it seems that the empty lines delineate the different  categories of dependencies ... might remove so that snyk runs again anyway :smile: